Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs

```diff
@@ -667,5 +667,10 @@ public virtual void SceneReady(Scene scene, LoadSceneMode mode)
 
 			}
 		}
-	}
+
+        public static GameObject GetGameObjectByNetworkId(uint id)
+        {
+            return ((NetworkBehavior)Instance.Networker.NetworkObjects[id].AttachedBehavior).gameObject;
```

Comment:
There should also be checks here that the `Networker` is actually been set. The `Instace` is not null. The `AttachedBehavior` is not null etc.

Reply:
Would it not be better to let it null error, so it is obvious that these are not set? Only one I would say to check would be `AttachedBehavior`.